### PR TITLE
Add kuadio ad network 

### PR DESCRIPTION
**Pull request by camel2243**
We added our ad network "kuadio".

* Add network logic
* Add samples and doc
* Update configuration files

Could you review our commits?

Best regards.

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -506,6 +506,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  kuadio: {
+    renderStartImplemented: true,
```
Comment (calebcordry) [inline ads/_config.js]: are you sure you are firing this method in your script? I couldn't get the listener to activate

**Comment by calebcordry** [comment]
Thanks for contributing! A few notes. Let me know if you have any questions

File: ads/_config.js (hunk 1 above)
Comment (camel2243) [reply ads/_config.js]: Guidelines suggests that renderStart API implementation is optional. 

Could We implement renderStart API in a near future?

**Comment by camel2243** [comment]
@calebcordry Thanks for you review.

I checked your review and I fixed the document.
I would really appreciated it, if you could check it again.

**Comment by camel2243** [comment]
Hello @calebcordry , we have made change fixes you requested but the travis job failed.
Is it related to our last changes or I must rebase master branch?

File: ads/_config.js (hunk 1 above)
Comment (calebcordry) [reply ads/_config.js]: It's true, it is up to you whether or not to implement. If you are not going to implement with this PR just remove `renderStartImplemented` and you can add it with a later PR when ready.

File: ads/kuadio.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    betaMode: data.betaMode === 'true',
+    debugMode: data.debugMode === 'true',
+    fastParse: data.fastParse !== 'false',
+  };
+
+  const e = global.document.createElement('div');
+  e.className = '_pvmax_recommend';
+  e.setAttribute('data-widget-id', data.widgetId);
+  e.setAttribute('data-ref', window.context.canonicalUrl);
+  global.document.getElementById('c').appendChild(e);
```
Comment (calebcordry) [inline ads/kuadio.js]: up to you, but there could potentially be naming collisions with your by using such a short and generic id

File: examples/ads.amp.html [hunk 3]
```diff
@@ -995,7 +995,7 @@ <h2>Kuadio</h2>
     type="kuadio"
     width="300"
     height="660"
-    layout="responsive",
+    layout="responsive"
```
Comment (calebcordry) [inline examples/ads.amp.html]: good catch 👍

File: ads/kuadio.md [hunk 4]
```diff
@@ -35,10 +35,11 @@ limitations under the License.
 
 For semantics of configuration , please contact [Kuadio](https://www.tenmax.io/kuadio)
 
-Supported parameters:
+Required parameters::
```
Comment (calebcordry) [inline ads/kuadio.md]: nit: only one :

File: ads/kuadio.md [hunk 5]
```diff
@@ -35,10 +35,11 @@ limitations under the License.
...
+- data-widget-id
 
+Optional parameters::
```
Comment (calebcordry) [inline ads/kuadio.md]: same as above

File: ads/kuadio.js (hunk 2 above)
Comment (camel2243) [reply ads/kuadio.js]: @calebcordry container with id "c" is default provided  by amphtml, Could I change it ?

Comment (calebcordry) [reply ads/kuadio.js]: Oops. No, sorry you are right. Disregard.

**Comment by camel2243** [comment]
@calebcordry Thank you for the review. I have already fixed typo and removed `renderStartImplemented` config.

File: ads/kuadio.js (hunk 2 above)
Comment (camel2243) [reply ads/kuadio.js]: 👌

**Comment by calebcordry** [comment]
This looks good. Are you ready for it to be merged?

**Comment by camel2243** [comment]
Yes we are ready, thanks @calebcordry  :)

**Comment by calebcordry** [comment]
Thanks for contributing!

**Comment by camel2243** [comment]
@calebcordry Sorry, I found a typo (https://github.com/ampproject/amphtml/pull/14096/files#diff-4ae7bcb2cdfa82facc881499d2d9ec62R41), it should be `https`.  I open a pr to fix it (https://github.com/ampproject/amphtml/pull/14630).

**Pull request merged**